src/mem/alloc.h

```diff
@@ -480,13 +492,13 @@ namespace snmalloc
     }
 
     template<Boundary location = Start>
-    static address_t external_address(void* p)
+    address_t external_uintptr(void* p)
```

Comment:
Why did you change the name back?  Looks like a merge conflict.

Reply:
'twas, indeed; thanks for the catch.